bundles/AdminBundle/src/Controller/Admin/IndexController.php

```diff
@@ -22,6 +22,7 @@
 use Pimcore\Bundle\AdminBundle\Controller\AdminController;
 use Pimcore\Bundle\AdminBundle\HttpFoundation\JsonResponse;
 use Pimcore\Bundle\AdminBundle\Security\CsrfProtectionHandler;
+use Pimcore\Bundle\StaticRoutesBundle\Model\Staticroute;
```

Comment:
refactor the [pimcore settings](https://github.com/pimcore/pimcore/blob/77a80f1a3c61bee064d15fcc1ece929ed0316904/bundles/AdminBundle/src/Controller/Admin/IndexController.php#L249) to bundle as this is just required in `staticroutes.js`